nix/vast/plugins/source.json

```
@@ -1,7 +1,7 @@
 {
   "url": "git@github.com:tenzir/vast-plugins",
   "ref": "main",
-  "rev": "7aece543105143f21fb2f65f49c1f684d04626ed",
+  "rev": "20234dc85667a5aa55a307e712c15eb9d846ab40",
   "submodules": true,
   "shallow": true
```

You should be able to kickstart CI with:
```suggestion
  "shallow": true,
  "allRefs": true
```

---

I've used your script from https://github.com/tenzir/vast/pull/2965 and the Nix CI runs work now.